content/04.results.md

```diff
@@ -112,5 +112,32 @@ A perfectly calibrated model would follow the dashed diagonal line.
 The blue line represents the predictions before calibration and the orange line shows predictions after calibration. 
 ](https://raw.githubusercontent.com/greenelab/snorkeling/master/figures/model_calibration_experiment/model_calibration.png){#fig:discriminative_model_calibration}
 
-## Reconstructing Hetionet
-place figure of number of new edges that can be added to hetionet as well as edges we can reconstruct using this method
+
+### Baseline Comparison
+Once our discriminator model is calibrated, we compared our model's ability to predict edges in our test set to a previosuly published baseline model [@tag:cocoscore].
+We report performance for both models in terms of AUROC and AUPR (Figure {@fig:cocoscore_comparison}).
+Across edge types our model shows comparable performance against the baseline in terms of AUROC.
+Regarding AUPR, our model shows hindered performance against the baseline.
+The exception for both cases is CtD where our model performs better than the baseline.
+
+![
+Comparion between our model and CoCoScore model [@tag:cocoscore].
+We report both model's performance in terms of area under the receiver operating curve (AUROC) and area under then precision recall curve (AUPR).
+Our model achieves comparable performance against CoCoScore in terms of AUROC.
+As for AUPR CoCoScore consistently outperforms our model except for compound treats disease (CtD). 
+](https://raw.githubusercontent.com/greenelab/snorkeling/master/figures/literature_models/model_comparison.png){#fig:cocoscore_comparison}
```

Comment:
For the figure legend is there a name for "our model"? For example, if someone else were to show this figure in their presentation, "Our Model" would make less sense.

The AUROC bar chart should have a y-axis from 0.5 to 1.